metaflow/metaflow_environment.py

```diff
@@ -76,21 +76,45 @@ def get_client_info(cls, flow_name, metadata):
         str : Information printed and returned to the user
         """
         return "Local environment"
+    
+    def get_s3_download_command(self, s3_path, local_path):
+        """
+
+        Parameters
+        ----------
+        s3_path : str
+            Path to S3 file
+        local_path : str
+            Path to download S3 file to
+        
+        Returns
+        -------
+        str : Command 
+        """
+
+        return (
+            "%s -c \"" % self._python()
```

Comment:
I doubt '\"' would work here given that we use `shlex` downstream for parsing the entry point.